### DISCUSSION TITLE
Severity: moderate

### DISCUSSION THREAD

**Discussion by wahoomv** [inertiajs/inertia#2113] [Help]
why I get this error 
axios  0.8.1 - 0.27.2
Severity: moderate
Axios Cross-Site Request Forgery Vulnerability - https://github.com/advisories/GHSA-wf5p-g6vw-rhxx
No fix available
node_modules/@inertiajs/inertia/node_modules/axios
  @inertiajs/inertia  *
  Depends on vulnerable versions of axios
  node_modules/@inertiajs/inertia

2 moderate severity vulnerabilities

Some issues need review, and may require choosing( my setup is node v22.11.0)

**Comment by RobertBoes** [comment]
The latest version of `@inertiajs/inertia` is `0.11.1`, which won't be updated anymore. So I'd suggest upgrading to Inertia 1.x, outlined in the docs here: https://inertiajs.com/upgrade-guide
